Documentation/common-problems.md

```diff
@@ -312,6 +313,21 @@ This directory is the `dataDirHostPath` setting in the cluster CRD and is typica
 To fix the issue you will need to delete all components of Rook and then delete the contents of `/var/lib/rook` (or the directory specified by `dataDirHostPath`) on each of the hosts in the cluster. 
 Then when the cluster CRD is applied to start a new cluster, the rook-operator should start all the pods as expected.
 
+# Node hangs after reboot
+
+## Symptoms
+* After issuing a `sudo reboot` command, node never returned online
+* Only power cycle helps
+
+## Solution
+Node needs to be [drained first](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/).
+```
+$ kubectl drain <node-name> --ignore-daemonsets --delete-local-data
+```
+After succeseful drain, node could be rebooted as usually. Because `drain` automatically cause `cordon` on node, it needs to be uncordoned once it's back online.
```

Comment:
`After the successful drain` <- typo in "successful"
Change `node could be` to `node can be`.
I would move the explanation of `drain` causing `cordon` before the command so the user completely knows what he "causes" to the node with the command.

The cordon part should be moved to it's own line because it is a separate action from the successful drain operation (would be good to reword it depending on the other changes written in this comment).